vnext/Microsoft.ReactNative/RedBoxHandler.idl

```diff
@@ -5,77 +5,96 @@ import "ReactNativeHost.idl";
   }
 
   [webhosthidden]
-  DOC_STRING("This object provides information about the error.  For JavaScript errors, a call stack is also provided.")
-  interface IRedBoxErrorInfo {
+  DOC_STRING("This object provides information about the error. For JavaScript errors, a call stack is also provided.")
+  interface IRedBoxErrorInfo
+  {
     DOC_STRING("The error message.")
     String Message { get; };
 
-    DOC_STRING("If the message was adjusted for formatting, or otherwise processed, this contains the message before those modifications")
+    DOC_STRING(
+      "If the message was adjusted for formatting, or otherwise processed, "
+      "this contains the message before those modifications.")
     String OriginalMessage { get; };
 
     DOC_STRING("An identifier for this error.")
     String Name { get; };
 
-    DOC_STRING("This will contain the component stack where the error occurred, which can help identify the component that is producing the error")
+    DOC_STRING(
+      "This will contain the component stack where the error occurred, "
+      "which can help identify the component that is producing the error.")
     String ComponentStack { get; };
 
-    DOC_STRING("This Id can be used in @IRedBoxHandler.UpdateError to identify which error is being updated.  For native errors, this is currently always `0`, and @IRedBoxHandler.UpdateError will never be called.")
+    DOC_STRING(
+      "This Id can be used in @IRedBoxHandler.UpdateError to identify which error is being updated. "
+      "For native errors, this is currently always `0`, and @IRedBoxHandler.UpdateError will never be called.")
     UInt32 Id { get; };
 
     DOC_STRING("For JavaScript errors, this will contain the call stack of where the error occurred.")
     IVectorView<IRedBoxErrorFrameInfo> Callstack { get; };
 
-    DOC_STRING("Provides access to extra data attached to the error.  Adding additional data to the errors is not yet part of the stable API.")
+    DOC_STRING(
+      "Provides access to extra data attached to the error. "
+      "Adding additional data to the errors is not yet part of the stable API.")
     IJSValueReader ExtraData { get; };
   }
 
   [webhosthidden]
 #ifdef USE_DOCSTRING
   [doc_string(
-    "`IRedBoxHandler` provides an extension point to allow custom error handling within the React instance.  This can be useful if you have an existing error reporting system that you want React errors to be reported to.  The default implementation of `RedBoxHandler` shows an error messages in a error screen that covers the whole application window.
+    "`IRedBoxHandler` provides an extension point to allow custom error handling within the React instance.
+This can be useful if you have an existing error reporting system that you want React errors to be reported to.
+The default implementation of `RedBoxHandler` shows an error messages in a error screen
+that covers the whole application window.
 
 -- Insert Screenshot here --
```

Comment:
note that we need the screenshot here :D

Reply:
Removed